client/src/main/scala/drt/client/components/FlightsWithSplitsTable.scala

```diff
@@ -390,18 +397,29 @@ object FlightTableRow {
     .configure(Reusability.shouldComponentUpdate)
     .build
 
-  private def gateOrStand(walkTimes: WalkTimes, defaultWalkTime: Long, flight: Arrival): VdomTagOf[Span] = {
+  private def gateOrStand(walkTimes: WalkTimes, defaultWalkTime: Long, flight: Arrival, coachTransfer: List[CoachTransfer], hasRedListOrigin: Boolean): VdomTagOf[Span] = {
     val walkTimeProvider: (Option[String], Option[String], Terminal) => String =
       walkTimes.walkTimeForArrival(defaultWalkTime)
     val gateOrStand = <.span(s"${flight.Gate.getOrElse("")} / ${flight.Stand.getOrElse("")}")
     val gateOrStandWithWalkTimes = Tippy.interactive(
-      <.span(walkTimeProvider(flight.Gate, flight.Stand, flight.Terminal)),
+      <.span(coachWalkTime(hasRedListOrigin, walkTimeProvider(flight.Gate, flight.Stand, flight.Terminal), flight, coachTransfer)),
       gateOrStand
     )
     val displayGatesOrStands = if (walkTimes.isEmpty) gateOrStand else <.span(gateOrStandWithWalkTimes)
     displayGatesOrStands
   }
 
+  def coachWalkTime(hasRedListOrigin: Boolean, walkTime: String, flight: Arrival, coachTransfer: List[CoachTransfer]) = {
```

Comment:
Could we either rename this function or call it only when there's a coach? Currently it reads as though it's always applying a coach time but it only applies one if there is one to be applied.